docker-compose.yml

```diff
@@ -26,6 +26,8 @@ services:
       - LOGO_S3_BUCKET=local-logo-bucket
       - FORMSG_SDK_MODE=development
       - BOUNCE_LIFE_SPAN=1800000
+      - AWS_ACCESS_KEY_ID=fakeKey
```

Comment:
Can we document these env vars in the readme?

Reply:
Haven't these always been in `DEPLOYMENT_SETUP.md` though?